src/components/Courses.jsx

```diff
@@ -1,8 +1,21 @@
 import {React, useState, useEffect} from 'react';
-import {Card, Row, Container, Accordion} from 'react-bootstrap';
+import {Card, Row, Container, Accordion, Button} from 'react-bootstrap';
 import Loading from './Loading';
 import CourseForm from './CourseForm';
 
+async function deleteCourse(code) {
```

Comment:
pretty sure you don't need the `async` tag here. Only if you use `await` somewhere in the function.

In this case, you are returning the `fetch` call, which will return a promise. And you are correctly using async/await at the call site in your `hancleClick` function below